dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -291,6 +292,10 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                 GUID taskClsid = details.TaskClsid();
                 THROW_HR_IF(E_INVALIDARG, taskClsid == GUID_NULL);
 
+                winrt::guid registeredClsid{ GUID_NULL };
```

Comment:
Create a pure RegisterActivator() API. It will greatly simplify this logic!

Reply:
As discussed the activators will be removed in the upcoming PR. Thanks.